src/components/NewLocationPage/SummaryStatsBlock/SummaryStatsBlock.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
ignore this whole component for now. as we chatted about, vaccinations will need to render separately, etc etc